### ISSUE TITLE
The proposed conformance arrangements don't support a "zero defects" policy

### ISSUE THREAD

**Issue by jasonjgw** [w3c/silver#561]
In a comment recorded in an earlier issue, the WCAG 3.0 draft was praised for moving away from the so-called "zero defects" strategy of WCAG 2.x, whereby a single instance of failing to satisfy a success criterion renders the entire content non-conforming at the relevant conformance level.

The working draft moves sufficiently far away from this approach that it now appears to be infeasible to prescribe a "zero defects" policy with respect to its requirements. However, there are cases in which organizations, regulators or other policy-makers may have good reason to specify "zero defects" as a requirement in relation to suitable subsets of the WCAG 3 guidelines. Examples that come to mind include safety-critical software, some educational applications, election systems, and some public services. These are only illustrations, and not intended to limit or prejudge the categories of web-based documents or applications that may be suitable for such a policy.

For instance, at the highest scoring level, the WCAG 3 draft only provides assurance that 95% of images have appropriate text alternatives, making it impossible to specify a "100% coverage" policy in terms of WCAG 3.

If we consider WCAG 3 as a tool with which policy-makers (in organizations or as government regulators) can specify and measure conformance with their accessibility-related policy objectives, then the current draft falls short of supporting legitimate policy goals that may reasonably be pursued.

Although the draft provides for "critical errors" designed to prevent content from conforming if it has shortcomings that interfere sufficiently with the user's ability to complete tasks, the application of this provision raises interpretive problems. For instance, consider a typical textbook that includes expository material, examples and exercises. Are the examples part of a "process"? If not, then no critical error can be introduced by failing to provide an appropriate text alternative for images in the examples. If they are taken to be part of a "process" (e.g., that of understanding the educational material), then it still isn't clear in what cases critical errors arise. Suppose an example (or an exercise, which is arguably more likely to be part of a "process") contains an image that lacks a textual alternative. If a discerning student could infer the content of the image from the context, then is the image "needed" to complete a process (this being the test of whether there's a critical error)? Is it material whether one thinks that a typical student (or an under-performing students in the relevant context) would be sufficiently discerning in this case?

I think it would be legitimate for a publisher, an education provider or a government to decide that it would be preferable to avoid the interpretive doubts and to specify "zero defects" as the conformance goal. If appropriate, trivial defects could be addressed in the policy (e.g., although they constitute non-conformance to WCAG 3 at the prescribed conformance level, they would not amount to an infringement of the policy itself, if they are corrected in a timely manner wen detected).

Thus, I think WCAG 3 should provide a mode of conformance in which suitable subsets of the requirements are assessed in a pass/fail manner, as in WCAG 2.x, such that a single defect can result in non-conformance of the content as a whole. I am not arguing here that this has to be the only mode of conformance, but rather that it should be among the available options which a policy-maker can prescribe by reference to and with the support of the Guidelines. Another way of thinking about the issue is that if a content/software provider believes "zero defects" conformance has been achieved with respect to an appropriate subset of the WCAG 3 requirements, the conformance scheme should make it possible to declare this in a conformance claim. As I understand the draft, this is currently not supported - I can maximally claim 95-100% for text equivalents, for example.

A further difficulty of the approach to conformance developed in the public working draft is that it depends rather heavily on how one interprets the applicability of "critical errors" - either a requirement scores according to the proportion of "correct" instances, or it scores 0 due to a critical error. Hence all of the interpretive difficulties associated with whether a given issue constitutes a critical error become crucially important in determining the conformance outcome. This is likely to be a major source of subjectivity, and a cause for disagreement among evaluators, in applying the proposal.

**Comment by alastc** [comment]
Comments from the sub-group:

Jeanne: Opposed to baking this into requirements. We need to focus on guidance first. When we get to conformance, we can see what works. 

Gregg: Conformance and enforcement are separate. We're getting into policy, we work out the measure of accessibility, provide reasonable metrics, and regulators work out the rest.

Wilco: Agree, doesn't belong in requirements. Quite a few aspects of conformance are integrated.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
